src/pretix/base/models/discount.py

```diff
@@ -171,6 +173,17 @@ class Discount(LoggedModel):
                     "access to sold-out quota will still receive the discount."),
     )
 
+    subevent_date_from = models.DateTimeField(
+        verbose_name=_("Available for dates from"),
```

Comment:
```suggestion
        verbose_name=pgettext_lazy("subevent", "Available for dates starting from"),
```
The word "date" in English is quite ambiguous, so we need to ensure e.g. German translaters will translate with "Termin" not "Datum". Also, we can make it clearer that we only look at the start of the event